### PR TITLE
add svm output gpu code

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

**Comment by piiswrong** [comment]
@sbodenstein  Could you verify this?

**Comment by piiswrong** [comment]
@yajiedesign Looks like the original author is busy.
Could you do the following and we can merge:
1. fix lint http://107.23.36.180:8080/job/mxnet/530/console
2. add consistency test following examples in tests/python/gpu/test_operator_gpu.py

**Comment by yajiedesign** [comment]
@piiswrong ok i am add gpu test.

File: tests/python/gpu/test_operator_gpu.py [hunk 1]
```diff
@@ -152,9 +165,24 @@ def test_embedding_with_type():
...
+    sym = mx.sym.SVMOutput(name='svmoutput', use_linear=True)
+    ctx_list = [{'ctx': mx.gpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float64}},
+                {'ctx': mx.gpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float32}},
+                {'ctx': mx.gpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float16}},
+                {'ctx': mx.cpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float64}},
+                {'ctx': mx.cpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float32}},
+                {'ctx': mx.cpu(0), 'svmoutput_data': (20, 10), 'type_dict': {'svmoutput_data': np.float16}}]
+    arg_params = {'svmoutput_data': np.random.randint(low=0, high=10, size=(20, 10))}
+    check_consistency(sym, ctx_list,
+                      grad_req={'svmoutput_data': 'null', 'svmoutput_label': 'null', 'svmoutput_weight': 'write'},
```
Comment (piiswrong) [inline tests/python/gpu/test_operator_gpu.py]: Why not propogate to data? So can't train lower layers with this?

Comment (piiswrong) [inline tests/python/gpu/test_operator_gpu.py]: also where does weight come from? I only see data and label in listarguments

**Comment by piiswrong** [comment]
@sbodenstein Could you help review this?

**Comment by piiswrong** [comment]
where is device_launch_parameters?

**Comment by piiswrong** [comment]
I don't see the file in this PR. Where is it?

**Comment by yajiedesign** [comment]
@piiswrong include cuda

**Comment by piiswrong** [comment]
Ok. Please change "" to <>

**Comment by yajiedesign** [comment]
@piiswrong ok

**Pull request closed without merging**